config/zones/NZ.yaml

```diff
@@ -3,6 +3,35 @@ bounding_box:
     - -47.638394
   - - 178.798828
     - -34.168835
+capacity:
+  biomass:
+    datetime: '2022-01-01'
+    source: Ember, Yearly electricity data
+    value: 140.0
+  coal:
+    datetime: '2022-01-01'
+    source: Ember, Yearly electricity data
+    value: 500.0
+  gas:
+    datetime: '2022-01-01'
+    source: Ember, Yearly electricity data
+    value: 1920.0
+  hydro:
+    datetime: '2022-01-01'
+    source: Ember, Yearly electricity data
+    value: 5440.0
+  solar:
+    datetime: '2022-01-01'
+    source: Ember, Yearly electricity data
+    value: 260.0
+  unknown:
+    datetime: '2022-01-01'
+    source: Ember, Yearly electricity data
+    value: 1190.0
```

Comment:
No geothermal split?
That would account for most of this

Reply:
oops my bad